Loadouts/blu_usmc_m16_desert.hpp

```diff
@@ -0,0 +1,259 @@
+    launchers[] = {USMC_MAT};
+    secondaryAttachments[] = {USMC_MAT_OPTIC};
+  };
+  class rhsusf_usmc_marpat_wd_javelin: Fic_Spotter {// MAT Spotter/Ammo Bearer
+    vest[] = { "rhsusf_spc_light" };
+    LAUNCHER_GEAR("B_Carryall_mcamo", USMC_MAT_MAG2)
+  };
+  class rhsusf_usmc_lar_marpat_wd_crewman : Fic_Soldier_Carbine{ // HAT Gunner
+    LAUNCHER_GEAR("B_Carryall_mcamo", WEST_HAT_MAG)
+    launchers[] = {WEST_HAT};
+  };
+  class rhsusf_usmc_lar_marpat_wd_combatcrewman : Fic_Spotter{// HAT Spotter
+    vest[] = {"rhsusf_spc_light"};
+    LAUNCHER_GEAR("B_Carryall_mcamo", WEST_HAT_MAG)
+  };
+  class rhsusf_usmc_marpat_wd_stinger : Fic_Soldier_Carbine {// SAM Gunner
+    vest[] = { "rhsusf_spc_light" };
+    LAUNCHER_GEAR("B_Carryall_mcamo", WEST_SAM_MAG)
+    launchers[] = {WEST_SAM};
+  };
+  class rhsusf_usmc_marpat_wd_grenadier: Fic_Soldier_Carbine {// Mortar Gunner
+    handguns[] = { WEST_PISTOL };
+    magazines[] += {WEST_PISTOL_MAG};
+    vest[] = { "rhsusf_spc_light" };
+    MORTAR_GEAR("B_Mortar_01_weapon_F")
+  };
+  class rhsusf_usmc_marpat_wd_gunner: Fic_Spotter {// Assistant Mortar
+    vest[] = { "rhsusf_spc_light" };
+    MORTAR_GEAR("B_Mortar_01_support_F")
+  };
+  class rhsusf_usmc_marpat_wd_helipilot {// Pilot
+    uniform[] = {"U_B_HeliPilotCoveralls"};
+    backpack[] = {"B_AssaultPack_blk"};
+    vest[] = {"V_TacVest_blk"};
+    headgear[] = {"rhsusf_hgu56p"};
+    weapons[] = {WEST_SMG};
+    magazines[] = {WEST_SMG_MAG,CREW_GRENADES};
+    backpackItems[] = {KEY_WEST,RADIO_LR};
+    items[] = {BASE_MEDICAL,WEST_BASE_TOOLS,WEST_LEADER_TOOLS,RADIO_MR};
+    linkedItems[] = {WEST_BASE_LINKED,WEST_LEADER_LINKED};
+  };
+  class rhsusf_usmc_marpat_wd_helicrew: rhsusf_usmc_marpat_wd_helipilot {// Pilot Crewchief
+    headgear[] = {"rhsusf_hgu56p_mask"};
+  };
+  class rhsusf_usmc_marpat_wd_combatcrewman: Fic_Soldier_Carbine {// Crewman Commander
+    vest[] = {"rhsusf_spc_crewman"};
+    headgear[] = {"rhsusf_cvc_helmet"};
+    backpack[] = {"B_Carryall_mcamo"};
+    magazines[] = {WEST_CARBINE_MAG,CREW_GRENADES};
+    backpackItems[] = {KEY_WEST,RADIO_LR};
+    linkedItems[] = {WEST_BASE_LINKED,WEST_LEADER_LINKED,BINOS};
+    items[] += {BASE_MEDICAL};
+  };
+  class rhsusf_usmc_marpat_wd_crewman: rhsusf_usmc_marpat_wd_combatcrewman {// Crewman Gunner
+    backpackItems[] = {RADIO_MR,KEY_WEST};
+    linkedItems[] = {WEST_BASE_LINKED,WEST_LEADER_LINKED};
+  };
+  class rhsusf_usmc_marpat_wd_engineer: rhsusf_usmc_marpat_wd_combatcrewman{// Crewman Driver
+    backpackItems[] += {"Toolkit"};
+  };
+  class Fic_eng: rhsusf_usmc_marpat_wd_crewman {
+    headgear[] = {"rhsusf_lwh_helmet_marpatd"};
+    items[] += {BASE_ENG};
+    backpackItems[] = {};
+  };
+  class rhsusf_usmc_marpat_wd_explosives: Fic_eng{// EOD Explodey guy
+    vest[] = { "rhsusf_spc_rifleman" };
+    //backpackItems[] += {BASE_EXP};    // uncomment to add explosives
+    //backpackItems[] += {BASE_MINE};   // uncomment to add mines
```

Comment:
I'll make one default but I'm probably going to give explosives to the Assaultmen as well, in real life the marines rarely use dedicated EOD.